### PR TITLE
How-to mechanics for specifying embedding models

### PR DESCRIPTION
**Pull request by Paul-Cornell**
See https://unstructured-53-embedding-2024-08-19.mintlify.app/api-reference/how-to/embedding

### CODE REVIEW

File: api-reference/how-to/embedding.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+specifying an _embedding model_ that is provided or used by an _embedding provider_. An [embedding model](https://python.langchain.com/v0.2/docs/concepts/#embedding-models) creates a complex mathematical representation 
+known as a _vector_ that captures inherent properties and relationships between data. (In this case, this data is the text that is extracted by Unstructured.) 
+These vector embeddings allow _large language models_ (LLMs) and _vector databases_ to more quickly and efficiently analyze and process these inherent properties and relationships. 
+
+To use the Ingest CLI or Ingest Python library to generate embeddings, do the following:
+
+1. Choose an embedding provider that you want to use from among the following allowed providers, and note the provider's ID:
+
+   - The provider ID `langchain-aws-bedrock` for [Amazon Bedrock](https://aws.amazon.com/bedrock/). [Learn more](https://python.langchain.com/v0.2/docs/integrations/text_embedding/bedrock/).
```
Comment (ninalopatina) [inline api-reference/how-to/embedding.mdx:22]: can we add the pip install info here? Or are you thinking that would go somewhere else?

**Review by ninalopatina — DISMISSED** [review]

File: api-reference/how-to/embedding.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+   - `octai` for [Octo AI](https://octo.ai/). [Learn more](https://octo.ai/docs/text-gen-solution/using-unstructured-io-for-embedding-documents).
+
+2. For the following embedding providers, you can choose the model that you want to to use. If you do choose a model, note the model's name:
+
+   - `langchain-huggingface`. [Choose a model](https://huggingface.co/models?other=embeddings), or use the default model [sentence-transformers/all-MiniLM-L6-v2](https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2).
+   - `langchain-openai`. [Choose a model](https://platform.openai.com/docs/guides/embeddings/embedding-models), or use the default model `text-embedding-ada-002`.
+   - `langchain-vertexai`. [Choose a model](https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/text-embeddings-api), or use the default model `textembedding-gecko@001`.
+   - `langchain-voyageai`.  [Choose a model](https://docs.voyageai.com/docs/embeddings). No default model is provided.
+   - `octai`. [Choose a model](https://octo.ai/blog/supercharge-rag-performance-using-octoai-and-unstructured-embeddings/), or use the default model `thenlper/gte-large`.
```
Comment (MKhalusova) [inline api-reference/how-to/embedding.mdx:35]: is `octai` correct? the provider is called OctoAI. Shouldn't it be `octoai`?

Comment (Paul-Cornell) [reply api-reference/how-to/embedding.mdx:35]: Thanks for the catch!

File: api-reference/how-to/embedding.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+   - The provider ID `langchain-aws-bedrock` for [Amazon Bedrock](https://aws.amazon.com/bedrock/). [Learn more](https://python.langchain.com/v0.2/docs/integrations/text_embedding/bedrock/).
+   - `langchain-huggingface` for [Hugging Face](https://huggingface.co/). [Learn more](https://python.langchain.com/v0.2/docs/integrations/text_embedding/huggingfacehub/).
+   - `langchain-openai` for [OpenAI](https://openai.com/). [Learn more](https://python.langchain.com/v0.2/docs/integrations/text_embedding/openai/).
+   - `langchain-vertexai` for [Google Vertex AI PaLM](https://cloud.google.com/vertex-ai/docs/generative-ai/learn/overview). [Learn more](https://python.langchain.com/v0.2/docs/integrations/text_embedding/google_vertex_ai_palm/).
+   - `langchain-voyageai` for [Voyage AI](https://www.voyageai.com/). [Learn more](https://python.langchain.com/v0.2/docs/integrations/text_embedding/voyageai/).
+   - `octai` for [Octo AI](https://octo.ai/). [Learn more](https://octo.ai/docs/text-gen-solution/using-unstructured-io-for-embedding-documents).
+
+2. For the following embedding providers, you can choose the model that you want to to use. If you do choose a model, note the model's name:
```
[the remark this answers is not in the document]
Comment (Paul-Cornell) [reply api-reference/how-to/embedding.mdx:29]: Thanks for the catch!

**Review by MKhalusova — APPROVED** [review]

**Pull request merged**